docs/kit/outreach/rub-bots.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Rub-bots
+---
+
+The Rub-bots are a set of physical robots constructed in July/August
+2021 to facilitate [Smallpeice 2021](/events/history/2021). For that competition, they were called the 'Forklift-bots' to keep with the theme of the game. After that, they became the 'rub-bots' - named to reflect the fact that they were designed to perfectly fit inside the 35L Really Useful Boxes ('RUBs')
```

Comment:
the 35L rubs are actually too narrow to fit a rub-bot, but they fit perfectly in the 18L ones
```suggestion
2021 to facilitate [Smallpeice 2021](/events/history/2021). For that competition, they were called the 'Forklift-bots' to keep with the theme of the game. After that, they became the 'rub-bots' - named to reflect the fact that they were designed to perfectly fit inside the 18L Really Useful Boxes ('RUBs')
```